### ISSUE TITLE
Maps navigation improvement

### ISSUE THREAD

**Issue by cubap** [CenterForDigitalHumanities/TPEN-NL#53]
consider:

* link to record.html for the item
* link to the background essay
* display thumbnail in popover

**Comment by thehabes** [comment]
Definitely, all easy and an improvement to UX.  Added this point to Newberry notes as a concern.  Let's maybe ask what they would hope to see in those pop-ups after clicking the dots.

Current demo : http://paleo.rerum.io/TPEN-NL/manuscriptMap.html

**Comment by thehabes** [comment]
New current demo https://centerfordigitalhumanities.github.io/Newberry-French-paleography/map

I think the only thing we missed the the thumbnail.  Their current one doesn't have one.  If we wanted to, all we would have to do is add a `thumbnail:URL` property to the Features in the json file.  Not sure we will do it though.
